composer.json

```diff
@@ -0,0 +1,58 @@
+{
+    "name": "superdesk/web-renderer",
+    "license": "proprietary",
+    "type": "project",
+    "autoload": {
+        "psr-0": {
```

Comment:
PSR-0 is depracated. PSR-4 is now recommended as an alternative. (http://www.php-fig.org/psr/psr-0/)